src/files2move2bin/Config.TRACS

```diff
@@ -77,14 +77,14 @@
 Depth = 300.0   # Double in micrometers
 
 # Width of the read-out strips
-Width = 25.0   # Double in micrometers
+Width = 500 #25.0   # Double in micrometers
```

Comment:
I agree, will delete the comments. I uploaded my working version where there are lots of things commented so I easily change the parameters (= don't forget the previous setting).